demux/demux_playlist.c

```diff
@@ -54,11 +66,34 @@ struct m_sub_options demux_playlist_conf = {
             {"lazy", DIR_LAZY},
             {"recursive", DIR_RECURSIVE},
             {"ignore", DIR_IGNORE})},
+        {"directory-filter-types",
+            OPT_STRINGLIST(directory_filter)},
```

Comment:
Would it be better to let this default to `video,audio,image`? I can't think of a scenario where loading junk files in a directory could possibly be useful for users.

In the future I'm also thinking about adding something like `--playlist-drop-unknown` that automatically removes unknown or broken files from the playlist upon loading. This should make opening directories much more useful. At the moment when I open a directory with videos and subtitles the subtitles are added to the playlist as well and just stay in the playlist which is rather annoying.

Reply:
> Would it be better to let this default to video,audio,image? I can't think of a scenario where loading junk files in a directory could possibly be useful for users.

I'm not going to change existing behavior of directory loading in this PR. Feel free to open new one with this change.

Reply:
changed now